clients/docs/ccloud.rst

```diff
@@ -0,0 +1,229 @@
+
+   .. code-block:: bash
+
+      ccloud kafka topic produce test1 --parse-key --delimiter ,
+
+.. Is there any output the user should verfiy when running the previous command or expected output they should see?
+
+#. Type a few messages, using a ``,`` as the separator between the message key
+   and value:
+
+   .. code-block:: bash
+
+      alice,{"count":0}
+      alice,{"count":1}
+      alice,{"count":2}
+
+#. When you are done, press ``Ctrl-C``.
+
+.. Is there a source code link for this one? I only saw two .sh files in the ccloud directory, but we have
+   basic consumer/producer and then Avro consumer/producer. So assuming we'll need 4 source code links in total for this page?
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+.. The below step orginally said "Confluent Confluent CLI Consumer" originally
+   I changed it to Confluent Cloud CLI consumer as I think that is what was meant?
+
+#. Run the :ref:`Confluent Cloud CLI consumer <ccloud_kafka_topic_consume>`,
+   reading messages from topic ``test1``, passing in the following arguments:
+
+   -  ``-b``: print all messages from the beginning of the topic -
+   - ``--print-key``: print key and value (by default, it only prints value)
+
+   .. code-block:: bash
+
+      ccloud kafka topic consume test1 -b --print-key
+
+   You should see the messages you typed in the previous step.
+
+   .. code:-block: bash
+
+       alice   {"count":0}
+       alice   {"count":1}
+       alice   {"count":2}
+
+#. When you are done, press ``CTRL-C``.
+
+.. Not sure the correct source code link. I only saw two .sh files in the ccloud directory, but we have
+   basic consumer/producer and then Avro consumer/producer. So assuming we'll need 4 source code links in total for this page?
+
+#. View the :devx-examples:`consumer code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ccloud/ccloud-example.sh`.
+
+
+Avro And Confluent Cloud Schema Registry
+----------------------------------------
+
+.. include:: includes/client-example-schema-registry-3.rst
+
+
+Produce Avro Records
+~~~~~~~~~~~~~~~~~~~~
+
+.. Should steps 1 through 3 below be under this  "Produce Avro Records" section or before it?
+
+#. As described in the `Confluent Cloud
+   quickstart
+   <https://docs.confluent.io/current/quickstart/cloud-quickstart/schema-registry.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   in the |ccloud| GUI, enable |ccloud| |sr| and create an API key and
+   secret to connect to it.
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      ccloud kafka topic create test2
+
+#. Create a file that has the schema of your message payload.
+
+   .. code-block:: bash
+
+      echo '{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' > schema.json
+
+.. should the user verify anything after running the following
+   command or is there specific output they should see after running the command?
+
```

Comment:
@ybyzek Should the user verify anything after running the following command or is it specific output they should see after running the command?

Reply:
Nothing to verify

Reply:
Gotcha